jwst/guider_cds/setup.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
```

Comment:
setup.py files are no longer used or needed for step packages. Just the `__init__.py`. So this should be removed.